apps/deploy-web/src/components/new-deployment/ManifestEdit.tsx

```diff
@@ -58,6 +59,38 @@ export const ManifestEdit: React.FunctionComponent<Props> = ({ editedManifest, s
   const sdlBuilderRef = useRef<SdlBuilderRefType>(null);
   const { minDeposit } = useChainParam();
 
+  const searchParams = useSearchParams();
+  const templateId = searchParams.get('templateId');
+
+  const fileUploadRef = useRef<HTMLInputElement>(null);
+
+  const propagateUploadedSdl = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (!event.target.files || event.target.files.length === 0) {
+      return; // No file selected, exit the function
```

Comment:
this is really clear from the if above. I see no sense in such comments.

```suggestion
      return;
```

If you still believe this is useful here's a much cleaner way
```typescript
const hasFileSelected = !!event.target.files?.length
if (!hasFileSelected)
```

Reply:
Good point!